README.md

```diff
@@ -443,13 +455,29 @@ postgres:
 
 | Field | Description | Scheme | Required |
 | ----- | ----------- | ------ | -------- |
-| description |  | string | Yes |
-| driver |  | string | Yes |
-| connection |  | string | Yes |
-| query |  | string | Yes |
-| results |  | int | Yes |
-
+| description | description for the test | string | No |
+| driver | sql driver | string | Yes |
+| connection | connection string to connect to the server | string | Yes |
+| query | query that needs to be executed on the server  | string | Yes |
+| results | Number of rows returned by query. Defaults to 1 | int | No |
+| resultsFunction |  resultsFunction defines the results that needs to be checked. | string | No |
+| displayTemplate | displayTemplate represents the output format for the results. When defined results are published in text format. | string | No |
+
+Example with resultsFunction:
+```yaml
+postgres:
+    - connection: "host=localhost port=5432 user=postgres password=password dbname=postgres sslmode=disable"
+      query: "SELECT * from names"
+      results: 2 # number of rows
+      driver: postgres
+      resultsFunction: '[[ if index .results 0 "surname" | eq "khandelwal" ]]true[[else]]false[[end]]'
+      displayTemplate: '[[ index .results 0 ]]'
+```
+This check has an option to display result in text format instead of pass/fail bar format.
+To enable text based result user would define `displayTemplate`.
```

Comment:
```suggestion
This check has an option to display result in text format instead of pass/fail bar format.

To enable text based result user would define `displayTemplate`.
```